### ISSUE TITLE
Issue with selecting NPCs after a server reload or restart.

### ISSUE THREAD

**Issue by ImAPorpoise** [CitizensDev/Citizens#901]
Hello,

I love the Citizens plugin and I've never had any issues with it. However just recently upon the update of my server to 1.2, everything goes great with NPC creating and usage up until I reload or restart my server. After that, I cannot select the NPCs by right clicking, and because of this the player can also not talk to the NPC. I have to use the /npc select command to select an NPC, but the best I can do after that is turn on /npc talkclose. I can't imagine the issue... 

I'm running Citizens 1.2 Dev Build 35, and the latest Bukkit 1.2 dev build.

I've checked my error log and there are no errors...

I am still on Citizens 1. Do I need to update to Citizens 2?

Thanks in advance.

**Comment by charlie1990** [comment]
No, Citizens 2 is incomplete and not meant for a server with traffic on it, If you go to their bukkit dev page they have a 1.2 DEV BUILD of Citizens that should fix this problem.

**Comment by ImAPorpoise** [comment]
"I'm running Citizens 1.2 Dev Build 35"....

Do you have a direct link to the build you're using?

**Comment by Sydael** [comment]
I noticed that when a player is online while npc is created, the player can interact with it. And it applies to the creator and any other person that is online on server while creation. But when that player logs out or isn't on while NPC is created, he cannot interact with it. Also the creator of NPC, once relogged after creating, he cannot interact with it either.
Reloading works, but only for people that are online. And even so, when they relog, the problem reappears.

**Comment by Martinius** [comment]
I have exactly the same problem but found a way to interact withe citzens anyway. you can use /npc select <id>
then you can modify a npc 
But please update citizens so it works normal again

**Comment by ghost** [comment]
This REALLY needs to be fixed.

**Comment by Sydael** [comment]
I really need a working blacksmith and healer. They just dont work

**Comment by damopewpew** [comment]
I also have this issue but the interaction issue seems to be to do with teleporting to the npc. If you walk away from them and then come back they're fine, yet I'm having issues with Trader not working. The interaction issue could be to do with a Entity interaction bug that bukkit apparently has

**Comment by quickclay** [comment]
Just to add to the mix, I'm having the issue with the dev build on 1.2.3 too.

**Comment by RedStoner** [comment]
confirmed on dev build 37 of citizens with beta 1.2.3 of bukkit.

**Comment by quickclay** [comment]
Does it happen in builds right before 37?

**Comment by charlie1990** [comment]
Confirmed as well, however my server has the added bonus of getting random errors pertaining to guards and quester errors when I try interacting with anything related to a quest

And yes it does quickclay

**Comment by quickclay** [comment]
This also means the Blacksmith, Quester, Wizard functions etc that require clicking on the Citzien don't work.

**Comment by funky-man** [comment]
same issue still

**Comment by sammy123x** [comment]
Can someone please solve this issue? #910

**Comment by bucktower** [comment]
yep, works on mine - this is critical.
